src/lib/cpd/CpdEngine.ts

```diff
@@ -216,10 +216,10 @@ export class CpdEngine extends AbstractRuleEngine {
 			for (const occ of occurences) {
 				// create a violation for each occurence of the code fragment
 				const violation: RuleViolation = {
-					line: occ.attributes.line as number,
-					column: occ.attributes.column as number,
-					endLine: occ.attributes.endline as number,
-					endColumn: occ.attributes.endcolumn as number,
+					line: Number(occ.attributes.line),
```

Comment:
The "as number" doesn't actually change the type it turns out if it is a string. So using "Number" here so that I can actually test it as a number.